lib/system/generic/xlnx_common/zynqmp_aarch64/sys.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2022, Xilinx Inc. and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * @file	generic/xlnx_common/zynqmp_aarch64/sys.c
+ * @brief	machine specific system primitives implementation.
+ */
+
+#include <metal/compiler.h>
+#include <metal/io.h>
+#include <metal/sys.h>
+#include <stdint.h>
+#include "xil_cache.h"
+#include "xil_exception.h"
+#include "xil_mmu.h"
+#include "xscugic.h"
+
+#if defined(versal)
+#include "xcpu_cortexa72.h"
+#else
+#include "xreg_cortexa53.h"
+#endif /* defined(versal) */
+
+#define MB (1024 * 1024UL)
+#define GB (1024 * 1024 * 1024UL)
```

Comment:
Seems that thes definitions are used in several files... could be moved to utilities.h, the update of the other files can be done later

Reply:
ok will move this. thank you